src/app/middleware/etagCache.js

```diff
@@ -1,14 +1,20 @@
 module.exports = (deps) => {
   const {
-    '/lib/bundler/buildHash': buildHash
+    '/lib/bundler/buildHash': buildHash,
+    '/config/index': getConfig
   } = deps;
 
-  const hash = buildHash();
+  const config = getConfig();
 
+  const hash = buildHash();
   const returnHash = () => hash;
 
   const etagRequest = (req, res, next) => {
     if(req.get('If-None-Match') === hash) {
+      if(config.newRelicEnabled) {
+        const newRelic = require('newrelic');
+        newRelic.setControllerName('ETAG 304 Not Modified');
```

Comment:
Were you thinking something like 

```
      if(config.newRelicEnabled) {
        const newRelic = require('newrelic');
        newRelic.setControllerName(`ETAG 304: ${req.url}`);
      }
```

I wonder how well this would group the requests?